roles/infra/roles/storages/tasks/main.yml

```diff
@@ -82,7 +82,7 @@
     auth: "{{ ovirt_auth }}"
     id: "{{ ansible_version.full is version('2.6.0', '>=') | ternary(item.1.id, item.1.id[0]) }}"
     storage: "{{ item.0.name }}"
-    address: "{{ storages[item.0.name][item.0.storage.type].address | default(omit) }}"
+    address: "{{ omit if item.0.storage.type == 'iscsi' or storages[item.0.name][item.0.storage.type].address | default(omit) }}"
```

Comment:
why is this a correct fix?

Reply:
as I wrote elsewhere, I wouldn't go with this change - the other properties are also probably incorrect
we apply the same properties from `storages` to all connections which is wrong

Reply:
This is a patch to be able to keep running but as @ahadas said maybe we need to create a better solution.

IMHO when we create storage domains as we doing in lines:
https://github.com/oVirt/ovirt-ansible-collection/blob/master/roles/infra/roles/storages/tasks/main.yml#L6-L66
we should have the option to update the storage_connections in the same module "ovirt_storage_domain" 
so we will not have to run the "Update storage parameters" as we have it in:
https://github.com/oVirt/ovirt-ansible-collection/blob/master/roles/infra/roles/storages/tasks/main.yml#L80-L102
@mnecas wdyt?

Reply:
That seems to me like an RFE and not a regression

Reply:
We didn't do any relevant changes ovirt_storage_connection for at least 2 years

Reply:
@mwperina maybe it's more like a bug as you can see the full description in:
https://bugzilla.redhat.com/show_bug.cgi?id=2079896

Reply:
@mwperina you're right when looking at it from the ansible perspective but due to changes on the vdsm side, we may get to a worse situation then we would have in the past because of this issue

Reply:
@avihaie that's debatable - from QE perspective it's an awful regression because the tests fail
but from user's perspective it might be better to have the host set to non-operational and know about this issue (and ideally fix it)
however, in this context of ansible-collection, this is something we can fix in the next release